.github/issue-labeler.yml

```diff
@@ -0,0 +1,146 @@
+assessment:
+  - assessment
+  - bonus
+  - complaint
+  - submission
+  - feedback
```

Comment:
This might lead to problems when we talk about "automatic feedback" from e.g. Athena